samples/estimation/df-chemistry/src/df_chemistry.qs

```diff
@@ -86,9 +86,9 @@ namespace Microsoft.Quantum.Applications.Chemistry {
     /// electron operators, which are computed from the double factorized
     /// problem and parameters.
     internal newtype DoubleFactorizedChemistryConstants = (
-        RotationAngleBitPrecision: Int,
-        StatePreparationBitPrecision: Int,
-        TargetError: Double
+        RotationAngleBitPrecision : Int,
```

Comment:
Note how this is indented correctly compared to a couple of the examples above. I wonder if it's the comments causing the issue. I notice this doesn't have comments on the line following the open parens, but a couple examples of it not indenting correctly above do have comments on the next line following the open paren.

Reply:
Indentation logic will be addressed by a future PR.